query-portal-logs.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import boto3
+import time
+from datetime import datetime, timedelta
+from csv import DictWriter
+
+
+def combine_field_and_value_to_item(d):
+    return {d['field']: d['value']}
+
+
+if __name__ == "__main__":
+    client = boto3.Session(profile_name='harvarddev').client('logs')
+
+    query = client.start_query(
+        logGroupName='portal-ui-logs',
+        startTime=int((datetime.now() - timedelta(days=7)).timestamp()),
+        endTime=int(datetime.now().timestamp()),
```

Comment:
Is this required, or could you just leave it off?

Reply:
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/logs.html#CloudWatchLogs.Client.start_query `endTime` is required.